Tests/TABResourceLoaderTests/Mocks/MockObject.swift

```diff
@@ -23,6 +23,6 @@ extension MockObject {
 
 extension MockObject: Equatable {}
 
-func ==(lhs: MockObject, rhs: MockObject) -> Bool {
+func == (lhs: MockObject, rhs: MockObject) -> Bool {
```

Comment:
Hmm, the convention for equatable isn't to have a space after `==`, right? Is that an oversight in the linting or expected?